projects/js-packages/publicize-components/src/components/panel/styles.module.scss

```
@@ -0,0 +1,10 @@
+.one-click-share-dropdown {
+	cursor: pointer;
+	position: absolute;
+	top: 13px;
+	right: 50px;
```

This worried me a bit! But I've played around with zooming text etc, and stays in the right position. Is there anything else this could cause problems with? RTL languages for instance?

---

I tried RTL with a chrome addon and looked okay for me.

---

To make it [RTL aware](https://www.w3.org/TR/css-logical-1/#inset-properties), you can use

```suggestion
	position: absolute;
	inset-block-start: 13px;
	inset-inline-end: 50px;
```

---

Good idea! Is there enough browser support for this? I think It goes back far enough looking at [caniuse](https://caniuse.com/css-logical-props).